apps/src/applab/designElements/RestoreThemeDefaultsButton.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import commonStyles from '../../commonStyles';
+import PropTypes from 'prop-types';
+import Radium from 'radium';
+
+const styles = {
+  restoreButton: {
+    backgroundColor: '#0aa',
+    color: 'white',
+    float: 'right'
+  }
+};
+
+/**
+ * A restore theme defaults button
+ */
+class RestoreThemeDefaultsButton extends React.Component {
+  static propTypes = {
+    handleRestore: PropTypes.func.isRequired
+  };
+
+  render() {
+    const {handleRestore} = this.props;
+    return (
+      <button
+        type="button"
+        style={[commonStyles.button, styles.restoreButton]}
```

Comment:
Nit: if you use the spread operator to combine styles I don't think you need Radium.
```suggestion
        style={...commonStyles.button, ...styles.restoreButton}
```

Reply:
Yes, I was using the pattern from `DuplicateElementButton` - which uses Radium. Not sure which the team prefers. (Coming from react-native, I find the array syntax to be nice, but I recognize that it is not built-in for react.js)

Reply:
> It looks like the width of the button is causing layout issues at all screen sizes smaller than ~1200px:

Ack, yeah this is a problem within our supported range. *Let's change the string to "Restore Defaults",* this is shorter than the "Copy to Screen" button, which is currently the longest:
<img width="1040" alt="Screen Shot 2019-06-26 at 2 19 26 PM" src="https://user-images.githubusercontent.com/1669271/60215857-73748380-981d-11e9-93af-96a7eab05e85.png">

Seems like the reflow in this container should be more intentionally-designed considering all the features that have accumulated in the last few years. We've talked about having Mark take a look at design mode to improve the UI a few times but never prioritized it. I'll get it added to his backlog to see if there's general layout/cleanup work wew would want to do here.